L6_Pipes/Server/Employee.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+#include <iostream>
+#include <fstream>
+#include <exception>
+
+struct Employee
+{
+	int num;
+	char name[10];
+	double hours;
+};
+
+inline std::ostream& operator<<(std::ostream& out, const Employee& e)
```

Comment:
Inline is not really necessary
as for me I prefer to_string(), it's not so efficient, but may be reused well